pkgs/amino/tests/common.go

```diff
@@ -332,29 +332,29 @@ type AminoMarshalerStruct7 struct {
 	A int8
 }
 
-func (ams AminoMarshalerStruct7) MarshalAmino() ([]ReprElem7, error) {
-	return []ReprElem7{{A: ams.A}}, nil
+func (goo AminoMarshalerStruct7) MarshalAmino() ([]ReprElem7, error) {
```

Comment:
`goo`?

Reply:
I believe the naming comes from [here](https://github.com/gnolang/gno/blob/master/pkgs/amino/genproto/bindings.go#L548) 🤷‍♂️ 

Regardless, I've standardized the receivers here:
[02ca1f2](https://github.com/gnolang/gno/pull/472/commits/02ca1f21246deda4dce8117b6303c5af2917938e)